bin/peekaboo-util.py

```diff
@@ -43,49 +90,27 @@ def main():
                         help='Output additional diagnostics')
     parser.add_argument('-s', '--socket_file', action='store', required=True,
                         help='Path to Peekaboo\'s socket file')
-    parser.add_argument('-f', '--filename', action='append', required=True,
-                        help='Path to the file to scan. Can be given more '
-                        'than once to scan multiple files.')
-    args = parser.parse_args()
-
-    result_regex = re.compile(r'.*wurde als',
-                              re.MULTILINE + re.DOTALL + re.UNICODE)
-    peekaboo = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
-    peekaboo.connect(args.socket_file)
 
-    file_snippets = []
-    for filename in args.filename:
-        file_snippets.append('{ "full_name": "%s" }' % path.abspath(filename))
-    request = '[ %s ]' % ', '.join(file_snippets)
+    scan_file_parser = subparsers.add_parser('scan_file',
```

Comment:
I'd vote for 'scan-file' as command name here. Looks less techie, more like end-user interface and corresponds to peekaboo_-_util. --socket_file may be another candidate for beautification. :)